contrib/bugzilla-submit/bugzilla-submit.xml

```diff
@@ -201,12 +201,12 @@ password field is the right password.  The URL in the machine field
 must be enclosed in double quotes.</para>
 
 <para>For example, if your Bugzilla instance is at
-"http://landfill.bugzilla.org/bztest/", and your login and password
+"https://bugzilla-dev.allizom.org/bztest/", and your login and password
```

Comment:
Since this is just an example, I thought leaving the mozilla server here would be fine. Or should we remove this paragraph too?

Reply:
I would keep the paragraph, because it is helpful, but change the URL everywhere to something like `https://bugzilla.example.org/`. This makes somewhat clear that nobody can rely on the mentioned system being actually available.

> The domain names example.com, example.net, example.org, and example.edu are second-level domain names in the Domain Name System of the Internet. They are reserved by the Internet Assigned Numbers Authority (IANA) at the direction of the Internet Engineering Task Force (IETF) as special-use domain names for documentation purposes. The domain names are used widely in books, tutorials, sample network configurations, and generally as examples for the use of domain names. The Internet Corporation for Assigned Names and Numbers (ICANN) operates web sites for these domains with content that reflects their purpose.

https://en.wikipedia.org/wiki/Example.com